src/js/webrequest.js

```diff
@@ -776,6 +784,16 @@ function dispatcher(request, sender, sendResponse) {
     badger.storage.getBadgerStorageObject("action_map").deleteItem(request.origin);
 
     sendResponse();
+  } else if (request.type == "saveErrorText") {
+    let activeTab = badger.tabData[request.tabId];
+    activeTab.errorText = request.errorText;
+
+    sendResponse();
```

Comment:
Do we need the `sendResponse`? We don't do anything on response to this message back in the popup.

Reply:
Certainly, it is more of a deformation of my coding, I tend to return anything always in a function.